dotCMS/src/main/java/com/dotcms/filters/interceptor/cas/CasAutoLoginWebInterceptor.java

```diff
@@ -35,7 +48,7 @@ public Result intercept(final HttpServletRequest request,
 
             if (UtilMethods.isSet(userID)) {
 
-                if (LoginFactory.doCookieLogin(PublicEncryptionFactory.encryptString
+                if (this.loginServiceAPI.doCookieLogin(PublicEncryptionFactory.encryptString
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "PublicEncryptionFactory"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)